src/sonic-yang-models/tests/yang_model_tests/tests/cbfmaps.json

```diff
@@ -0,0 +1,24 @@
+{
+  "DSCP_TO_FC_MAP_CREATE": {
+    "desc": "Configure a DSCP to Forwarding class map."
+  },
+  "DSCP_TO_FC_MAP_CREATE_INVALID_DSCP": {
+    "desc": "Configure a DSCP to Forwarding class map with invalid key.",
+    "eStr": "Invalid DSCP"
```

Comment:
eStr should match to the error from Libyang. This may not pass. Do you see successful build of PKG.

Reply:
`                list DSCP_TO_FC_MAP { //this is list inside list for storing mapping between two fields

                    key "dscp";

                    leaf dscp {
                        type string {
                            pattern "6[0-3]|[1-5][0-9]?|[0-9]?" {
                                error-message "Invalid DSCP";
                                error-app-tag dscp-invalid;
                            }
                        }
                    }`
Unless I misunderstood what you meant I think the error strings do match in this case.
Can you please expand on what I need to run to build PKG in order to check?(I'd expect it to be in order since the pipeline checks have passed)